examples/helloworld/helloworld/helloworld.proto

```diff
@@ -15,9 +15,6 @@
 syntax = "proto3";
 
 option go_package = "google.golang.org/grpc/examples/helloworld/helloworld";
-option java_multiple_files = true;
```

Comment:
I at first thought this was to keep in sync with https://github.com/grpc/grpc-proto/blob/master/grpc/examples/helloworld.proto, but then I noticed the grpc-proto version doesn't have `go_package`. So I think this change is OK.

Reply:
I don't see a reason why we should have a `helloworld.proto` in our repo. Why can't we use the one in `grpc-proto` repo?

Reply:
Also, I don't see a proto for the `RouteGuide` or the `Echo` examples in grpc-proto and each of the repos have their own versions of these protos. We should address that as well.

Reply:
Also, we seem to own a copy of the `reflection.proto`. Shouldn't we be using it from `grpc-proto` repo?

Reply:
> I at first thought this was to keep in sync with https://github.com/grpc/grpc-proto/blob/master/grpc/examples/helloworld.proto, but then I noticed the grpc-proto version doesn't have `go_package`. So I think this change is OK.

Thanks for your review! @menghanl

Reply:
> Also, I don't see a proto for the `RouteGuide` or the `Echo` examples in grpc-proto and each of the repos have their own versions of these protos. We should address that as well.

For the moment, such is the case. So we can remove the needless java option in the go example.

Reply:
We should pursue what @easwars is suggesting instead of making changes here, since we would intend to delete these proto files anyway.